### PR TITLE
[Component with its own class name as service type ] LPS-196226 Refactor Registry classes from multiple modules

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-196226

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by MrJohn1911** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/JSBundleConfigRegistryUtil.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+			if (jsConfig != null) {
+				_bundleContext.ungetService(serviceReference);
+
+				_lastModified = System.currentTimeMillis();
+			}
+		}
+
+	}
+
+	{
```
Comment (rafaprax) [inline modules/apps/frontend-js/frontend-js-bundle-config-extender/src/main/java/com/liferay/frontend/js/bundle/config/extender/internal/JSBundleConfigRegistryUtil.java:131]: it must be a block static

File: modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/PanelCategoryRegistryUtil.java [hunk 2]
```diff
@@ -150,35 +147,31 @@ public PanelCategory getPanelCategory(String panelCategoryKey) {
...
+		_panelCategoryServiceTrackerMap;
+
+	static {
+		Bundle bundle = FrameworkUtil.getBundle(
+			PanelCategoryRegistryUtil.class);
+
 		_childPanelCategoriesServiceTrackerMap =
 			ServiceTrackerMapFactory.openMultiValueMap(
-				bundleContext, PanelCategory.class, null,
+				bundle.getBundleContext(), PanelCategory.class, null,
```
Comment (rafaprax) [inline modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/PanelCategoryRegistryUtil.java:164]: extract to a variable.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Pull request closed without merging**